fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -3400,4 +3400,14 @@ public PayloadPersistenceResponse storePayload(Resource resource, String logical
         // Delegate to the persistence layer. Result will be null if offloading is not supported
         return persistence.storePayload(resource, logicalId, newVersionNumber, resourcePayloadKey);
     }
+    
+    @Override
+    public List<Issue> validateDeleteResource(String type, String id) throws FHIROperationException {
```

Comment:
I was expecting to see logic like this within ValidateOperation.java (rather than a new method on FHIRRestHelper).  Do you expect it to be used from anywhere else?

Reply:
Its a little ugly (requires a cast) but it is possible to get the FHIRPersistence implementation straight from your operation impl by using something like this:
```
FHIRPersistence pl =
                    (FHIRPersistence) operationContext.getProperty(FHIROperationContext.PROPNAME_PERSISTENCE_IMPL);
```

Reply:
this has been fixed. The logic has been moved to ValidateOperation.java as suggested.